spec/poros/food_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+
+RSpec.describe Food do
+  it "exists", :vcr do
+    food = Food.new(gtin_upc: 123, description: "coffee", brand_owner: "Coffee Co.", ingredients: "coffee")
+
+    expect(food).to be_an_instance_of(Food)
```

Comment:
Here I would expect to see assertions about the exact value of the attributes. Because you're stubbing the food data above, you also don't need to be worried about that data changing like it could with VCR so you could really test that each attribute is what it should be